client-network/src/network_ops/default_network_ops_client.rs

```diff
@@ -305,22 +305,24 @@ mod tests {
         let fee_algorithm = UnitFeeAlgorithm::default();
 
         let wallet_client = DefaultWalletClient::builder()
-            .with_wallet(storage)
+            .with_wallet(storage.clone())
+            .build()
+            .unwrap();
+        let wallet_client2 = DefaultWalletClient::builder()
```

Comment:
I don't this there is a need for creating multiple instances of same object here.

Reply:
lifetime management is tricky here,
i'll find a more nice way.